React/wbooks/src/redux/logout/actions.js

```diff
@@ -0,0 +1,14 @@
+import api from '../../Config/api';
```

Comment:
It's not necessary to make an actionCreator per "action". You should have an Auth reducer. In that reducer, you should have the login, logout, recover password, and any other action related to authentication.